consensus/istanbul/backend/backend.go

```diff
@@ -316,3 +352,87 @@ func (sb *backend) HasBadProposal(hash common.Hash) bool {
 func (sb *backend) Close() error {
 	return nil
 }
+
+// IsQIBFTConsensus returns whether qbft consensus should be used
+func (sb *backend) IsQIBFTConsensus() bool {
```

Comment:
by looking to this code, is the same as using ``IsQIBFTConsensusForHeader(sb.chain.CurrentHeader())` no? to avoid duplication

Reply:
`IsQIBFTConsensusForHeader` checks if the qbft consensus was enabled for a given height (header), where as `IsQIBFTConsensus()` checks if the qbft consensus is enabled for the current height/block

Reply:
non-blocking: ok, there are 2 differences: `sb.chain.CurrentHeader()` and not checking for `sb.qibftConsensusEnabled`. is there a way that we can re-use the same? as the rest is similar.

Reply:
I pushed a commit, that should re-use same code between the two methods